Source/p_pspr.c

```diff
@@ -737,7 +753,7 @@ void A_FireOldBFG(player_t *player, pspdef_t *psp)
 
   if (weapon_recoil && !(player->mo->flags & MF_NOCLIP))
     P_Thrust(player, ANG180 + player->mo->angle,
-	     512*recoil_values[wp_plasma]);
+	     512*recoil_values[wp_plasma].thrust);
```

Comment:
Any chance to implement recoil pitch for the "old" BFG similar to the plasma gun?

Reply:
Hm, I tried in https://github.com/fabiangreffrath/woof/pull/506/commits/d095cbc33ce22de8928e0b66741a076cce259773 , but it's not satisfying.

Reply:
> Hm, I tried in [d095cbc](https://github.com/fabiangreffrath/woof/commit/d095cbc33ce22de8928e0b66741a076cce259773) , but it's not satisfying.

How about this?

```patch
--- a/Source/p_pspr.c
+++ b/Source/p_pspr.c
@@ -757,7 +757,7 @@ void A_FireOldBFG(player_t *player, pspdef_t *psp)
     P_Thrust(player, ANG180 + player->mo->angle,
             512*recoil_values[wp_plasma].thrust);
 
-  if (weapon_recoilpitch)
+  if (weapon_recoilpitch && (leveltime & 2))
   {
     player->recoilpitch = recoil_values[wp_plasma].pitch;
   }
```

Reply:
This is better! Added in https://github.com/fabiangreffrath/woof/pull/506/commits/ff713ebf51453e6dc4d0d528d8fb144c8b5598c6